app-backend/database/src/main/scala/com/azavea/rf/database/tables/Projects.scala

```diff
@@ -46,6 +45,9 @@ class Projects(_tableTag: Tag) extends Table[Project](_tableTag, "projects")
   val tags: Rep[List[String]] = column[List[String]]("tags", O.Length(2147483647,varying=false), O.Default(List.empty))
   val extent: Rep[Option[Projected[Geometry]]] = column[Option[Projected[Geometry]]]("extent", O.Length(2147483647,varying=false), O.Default(None))
   val manualOrder: Rep[Boolean] = column[Boolean]("manual_order", O.Default(true))
+  val isAOIProject: Rep[Boolean] = column[Boolean]("is_aoi_project", O.Default(false))
+  val aoiCadence: Rep[Long] = column[Long]("aoi_cadence")
```

Comment:
I think this should include the unit in the name (e.g. `aoiCadenceMS` or `aoiCadenceMs`) so that consumers of the API do not need to look at documentation to interpret the value.